project.yaml

```
@@ -39,3 +39,9 @@ actions:
     outputs:
       highly_sensitive:
         cohort: output/dm/dm017_milan.csv.gz
+  generate_dep_reg_dataset_jaidip:
+    run: ehrql:v1 generate-dataset analysis/dep_reg_dataset_jaidip.py --output output/dep/dep_dataset_jaidip.csv.gz
+    outputs:
+      highly_sensitive:
+        dataset: output/dep/dep_dataset_jaidip.csv.gz
```

I dont think we need the name as a suffix anymore because there shouldn't be any conflicts with the names as each of you picked a different new register. I'd also try to stay close to the original naming of the QOF indicators, but that's just my preference so it's easier for me to find what I'm looking for:

```suggestion
  generate_dep_reg_dataset:
    run: ehrql:v1 generate-dataset analysis/dep_reg_dataset.py --output output/dep/dep_reg_dataset.csv.gz
    outputs:
      highly_sensitive:
        dataset: output/dep/dep_reg_dataset.csv.gz
```